### DISCUSSION TITLE
Welcome to UKBinCollectionData Discussions!

### DISCUSSION THREAD

**Discussion by robbrad** [robbrad/UKBinCollectionData#61] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dp247** [comment]
> Hi, I need help in setting up the integration, it asked me for my council and UPRN, which i inputed, but after going into the service and clicking configure i get this box and im not sure what to put in there, anyone know what I need to do. Thanks in advance. :)

Probably best to open a new issue for this

**Comment by njdthomas** [comment]
Hello.  First up... this is a great little app that's been made for Home Assistant - no longer do I need to trawl a clunky council website to learn what bin to put out each week! 

But I have a problem that I would welcome some help with please.

I'm adding a third property from Mole Valley and am faced with ...

[14 devices]
[50 entities]
Failed setup, will retry: Unexpected error: No collection data found for UPRN provided.

The first two properties setup just fine without any errors.  The council website has all the relevant data for the PRN so not sure where the problem lies.  What can I do to help troubleshoot this please?

**Reply by dp247** [reply]
If you haven't done so already, open a new issue and include the relevant details and HA logs, then we can take a look as to why you're getting that error 😊

**Comment by s9550110** [comment]
Just wanted to say thank you. We're coming up to Christmas and my bin collection has temporarily moved a day but my integration is seamless and notifications will just happen.

**Comment by scalolli** [comment]
Hey folks, 
I am currently interested in contributing to the project and quite happy to take up this council request issue: https://github.com/robbrad/UKBinCollectionData/issues/943

Could someone be kind enough to please assign this to me please? I was unable to assign this to myself. I am long term JVM developer but have been dabbling my hands with python recently so quite happy to see that this project is a nice little way for me to learn Python and provide some support to this project as well.

**Reply by dp247** [reply]
Done and done!
